### DISCUSSION TITLE
GitHub missing picture support

### DISCUSSION THREAD

**Discussion by iMonZ** [community/community#3149] [Discussions] [answered]
GitHub is missing this following picture formats:

.heif (image/heif)
and
.heic (image/heic)

As seen here:
https://mimetype.io/image/heic/amp
http://fileformats.archiveteam.org/wiki/HEIF

Please add this file types for a better user experience.

Currently the only option is to upload this files as an zip archive before uploading.

And if you download this files you can view them perfectly on a pc or phone
This is the default image format on some phones or computers after you create it with the camera or screenshot

nearly every computer (except for vintage ones) can open this kind of file types

Stay healthy!
reg

**Comment by evi-liu** [comment]
This space is currently reserved for feedback on Discussions and GitHub for Mobile. This is noted in our [README](https://github.com/github/feedback#github-public-feedback-discussions) and the [pinned discussion](https://github.com/github/feedback/discussions/1).

All other feedback at this time can be shared through our [support channel](https://support.github.com/contact/feedback) or our [community forum](https://github.community/).

**Comment by iMonZ** [comment, marked as answer]
Closed
